infer/lib/rust/tests/programs/operands/move/move_from_tuple.sil

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) Facebook, Inc. and its affiliates. 
+//
+// This source code is licensed under the MIT license found in the 
+// LICENSE file in the root directory of this source tree. 
+
+.source_language = "rust"
+
+define test() : void {
+  local var_0: void, var_1: (string, int), var_2: string, var_3: string
```

Comment:
There are no tuples in textual, you can redefine it as a stuct: type tuple_string_int = {0: string, 1: int)